purchasing/conductor/forms.py

```diff
@@ -47,6 +50,13 @@ def validate_multiple_emails(form, field):
             elif not re.match(EMAIL_REGEX, email):
                 raise ValidationError('One of the supplied emails is invalid!')
 
+def get_default():
+    return pytz.UTC.localize(
```

Comment:
This says, take the UTC now, add the UTC timezone to it, and then localize it to whatever the configured `DISPLAY_TIMEZONE` is (in our case, `US/Eastern`)